pyrogram/types/user_and_chats/chat_invite_link.py

```diff
@@ -104,11 +104,12 @@ def _parse(
         invite: "raw.base.ExportedChatInvite",
         users: Dict[int, "raw.types.User"] = None
     ) -> "ChatInviteLink":
-        creator = (
-            types.User._parse(client, users[invite.admin_id])
-            if users is not None
-            else None
-        )
+        if users is not None:
+            if not hasattr(invite, "admin_id"):
+                return None
+            creator = types.User._parse(client, users[invite.admin_id])
```

Comment:
```python
        creator = (
            types.User._parse(client, users[invite.admin_id])
            if users is not None and hasattr(invite, "admin_id")
            else None
        )
```
is better

Reply:
This shouldn't be the case because the invite instance is empty.

Reply:
```if not hasattr(invite, "admin_id"):
    return None
if users is not None:
    creator = types.User._parse(client, users[invite.admin_id])
else:
    creator = None```

Reply:
yes 

'ChatInvitePublicJoinRequests' object has no attribute 'link'
Traceback (most recent call last):
  File "venv/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 214, in handler_worker
    await parser(update, users, chats)
  File "venv/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 126, in chat_join_request_parser
    pyrogram.types.ChatJoinRequest._parse(self.client, update, users, chats),
  File "venv/lib/python3.10/site-packages/pyrogram/types/user_and_chats/chat_join_request.py", line 81, in _parse
    invite_link=types.ChatInviteLink._parse(client, update.invite, users),
  File "venv/lib/python3.10/site-packages/pyrogram/types/user_and_chats/chat_invite_link.py", line 116, in _parse
    invite_link=invite.link,
AttributeError: 'ChatInvitePublicJoinRequests' object has no attribute 'link'